src/com/company/Jdbc.java

```diff
@@ -0,0 +1,45 @@
+package com.company;
+
+import java.sql.*;
+
+import static com.company.Main.statement;
+
+public class Jdbc {
+
+    public static void databaseFailure(String txtFilePath,String searchWord,String errorMessage) throws SQLException {
+        Connection con = DriverManager.getConnection("jdbc:mysql://localhost:3306/elixr_intern_poc_01","vikas","root");
+        String query = "insert into audit values(?,?,now(),?,?,?)";
+        statement =con.prepareStatement(query);
+        statement.setString(1,txtFilePath);
+        statement.setString(2,searchWord);
+        statement.setString(3,"Failure");
+        statement.setInt(4,0);
+        statement.setString(5,errorMessage);
+        statement.executeUpdate();
+    }
+    public static void databaseSuccess(String txtFilePath,String searchWord,int count) throws SQLException {
```

Comment:
the exceptions are handled using try catch blocks
and in case DB is not reachable the exception will be handled and the stacktrace will be printed.
is that fine??